src/App.tsx

```diff
@@ -15,36 +17,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  length = 'length',
+  alph = 'alph',
+  default = '',
+}
+
+export const App = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.default);
+
+  const reverseList = () => {
+    if (isReversed) {
+      setIsReversed(false);
+    } else {
+      setIsReversed(true);
+    }
```

Comment:
```suggestion
    setIsReversed(currentState => !currentState);
```
Could be simplified and when you set state depending on the previous one use functional `setState`